pkg/csi/controller/utils.go

```diff
@@ -159,3 +168,14 @@ func selectDrive(ctx context.Context, req *csi.CreateVolumeRequest) (*types.Driv
 
 	return &maxFreeCapacityDrives[n.Int64()], nil
 }
+
+func validateVolumeClaimID(claimID string) error {
```

Comment:
This function is not needed. If value is matching `volumeClaimIDRegex`, the value is compatible to `LabelKey` because of it's format.